addons/crm/data/ir_action_data.xml

```diff
@@ -10,7 +10,7 @@
         <field name="binding_model_id" ref="crm.model_crm_lead"/>
         <field name="binding_view_types">list</field>
         <field name="state">code</field>
-        <field name="code">action = model.action_set_lost_with_reason()</field>
+        <field name="code">action = records.action_set_lost_with_reason()</field>
```

Comment:
Looks like most occurrences of this do "if records" beforehand.

Reply:
Yeah I noticed but it doesn't seem useful since `class IrActionServer`  ensures we at least have an empty recordset in `_get_eval_context` and `run` here: https://github.com/odoo-dev/odoo/blob/master-crm-lost-wizard-lead-ids-field-reth/odoo/addons/base/models/ir_actions.py#L618 

There's also a lot of places where it's not used too with `action = records.` so I think it should be ok.

Reply:
Note that au pire it opens the action without leads. Method won't crash on an empty recordset.